requirements/requirements.txt

```diff
@@ -3,7 +3,7 @@ ovos-utils~=0.0.35
 ovos-config~=0.0.10
 phoneme-guesser~=0.1
 ovos-plugin-manager~=0.0.24
-neon-utils[network]~=1.8
+neon-utils[network]~=1.8.3a5
```

Comment:
This should be:
```
neon-utils[network]~=1.8,>=1.8.3a5
```
The current spec (`~=1.8.3a5`) would not allow for 1.8.4+, the proposed change allows any compatible version (since we're using semver)